internal/clients/composer/client.go

```diff
@@ -33,7 +33,7 @@ var contentHeaders = map[string]string{"Content-Type": "application/json"}
 
 func NewClient(conf ComposerClientConfig) (*ComposerClient, error) {
 	if conf.URL == "" {
-		logrus.Warn("Composer URL not set, client will fail")
+		slog.Warn("composer URL not set, client will fail")
```

Comment:
Isn't log message supposed to be Upper cased?

Reply:
I propose to make all logging lower case, but good point: I will starte a thread on Slack before we merge this.